models/cam-fv/shell_scripts/synth_obs_locs_to_seqs.csh

```diff
@@ -0,0 +1,147 @@
+#!/bin/tcsh
+#
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# https://www2.cisl.ucar.edu/software/dart/download
+
+# ==============================================================================
+# Call create_obs_sequence to convert a text file of obs locations and types
+# into a set_def.out file.  
+# Call create_fixed_network_seq to create separate files for each time period. 
+# This one makes multiple files/day (2 or 4 are the most common), 
+# a single time per file.
+
```

Comment:
Add
# It requires only a few seconds to run a single date, 
# in which case it can be run interactively and the batch job directives can be ignored.